php/Terminus/Helpers/UpdateHelper.php

```diff
@@ -17,11 +17,13 @@ class UpdateHelper extends TerminusHelper
     public function getCurrentVersion()
     {
         $request  = new Request();
-        $url      = 'https://api.github.com/repos/pantheon-systems/terminus/releases';
-        $url     .= '?per_page=1';
-        $response = $request->request($url, ['absolute_url' => true]);
-        $release  = array_shift($response['data']);
-        $cache    = new FileCache();
+        $url = 'https://api.github.com/repos/pantheon-systems/terminus/releases/tags/0.x?per_page=1';
```

Comment:
I am suggesting the the `/releases/latest` will always return 0.x (the latest release) until we make a 1.0.0 release tag that **is not** marked pre-release. I believe this is the behaviour we want no?